### PR TITLE
[11.x] Adds id to inputs of the bug report page

### PR DESCRIPTION
**Pull request by milwad-dev**
This pull request includes the addition of an ID attribute to the inputs of bug reports.

## Why

You can populate input values using a parameter string in the URL, for example: `&laravel_version=11.0.1`. This pull request depends on https://github.com/laravel/framework/pull/50952.

### CODE REVIEW

File: tests/Integration/Foundation/Console/AboutCommandTest.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ public function testItCanDisplayAboutCommandAsJson()
             Assert::assertArraySubset([
                 'application_name' => 'Laravel',
                 'php_version' => PHP_VERSION,
-                'environment' => 'testing',
+                'environment' => 'workbench',
```
Comment (plumthedev) [inline tests/Integration/Foundation/Console/AboutCommandTest.php:20]: Why did you changed that?

**Comment by plumthedev** [comment]
This pull request is depend on the #50952 but in my opinion they could be merged in to one PR.
They are about one thing.

File: tests/Integration/Foundation/Console/AboutCommandTest.php (hunk 1 above)
Comment (milwad-dev) [reply tests/Integration/Foundation/Console/AboutCommandTest.php:20]: I will fix it

**Comment by milwad-dev** [comment]
@driesvints 
Why did tests fail?

**Pull request closed without merging**